ios-training-bjung/HomeViewController.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  ViewController.swift
+//  ios-training-bjung
+//
+//  Created by 鄭 普勤 on 2023/11/29.
+//
+
+import UIKit
+import YumemiWeather
+
+class HomeViewController: UIViewController {
+    
+    @IBOutlet weak var imageView: UIImageView!
+    
+    private var presenter: HomePresenterInput!
```

Comment:
追加したinitでpresenterがあることを確定できるので
```suggestion
    private var presenter: HomePresenterInput
```

Reply:
@novr 
5f6cd153371c74550ff09c88b5759584a19489aa
上記のコミットで対応しました。